APITaxi2/tasks/stats.py

```diff
@@ -112,10 +112,7 @@ def store_active_taxis(last_update):
             User,
             VehicleDescription.added_by_id == User.id
         ).options(
-            joinedload(Taxi.ads)
-            .joinedload(ADS.zupc)
-            .joinedload(ZUPC.parent)
-        ).options(
+            joinedload(Taxi.ads).joinedload(ADS.zupc).joinedload(ZUPC.parent),
```

Comment:
I see two different formatting styles with the hunk above. I don't have a preference.